### ISSUE TITLE
Provide support for separate aria label for the customButton on quickpick component

### ISSUE THREAD

**Issue by neeleshb** [microsoft/vscode#145768]
The quick pick component has only one aria-label that gets associated with the input box.
https://github.com/microsoft/vscode/blob/main/src/vs/base/parts/quickinput/browser/quickInput.ts#L440

The quickpick component also has a customButton which does not have custom aria-label.
https://github.com/microsoft/vscode/blob/main/src/vs/base/parts/quickinput/browser/quickInput.ts#L469

It would be nice to have a separate aria-label for this button.
